config/utils/logger.js

```diff
@@ -1,18 +1,29 @@
-/* eslint no-console: 0 */
+/* eslint-disable no-console */
 
 import chalk from 'chalk';
 
+const stripNewline = str => (
+    str.constructor === String
```

Comment:
I updated condition to `![null, undefined].includes(str) && str.constructor === String`